src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```
better choose map implementation, because just Fruit object should store information about that fruit (name, nutritions, country where it comes from) 
Product object would be better, but to make it easier and simplier lets store data in key-value pairs